### PR TITLE
adds example for nested doc query

### PR DESCRIPTION
**Pull request by terakilobyte**
This addresses an issue raised in https://jira.mongodb.org/browse/DOCS-9486, no nested document query example.

I had to adjust the example slightly in the quick-start. Can modify more as needed, I just want to get the ball rolling.

### CODE REVIEW

File: docs/quick-start.txt [hunk 1]
```diff
@@ -78,23 +78,34 @@ To insert a single document into a collection, use the
   client = Mongo::Client.new('mongodb://127.0.0.1:27017/test')
 
   collection = client[:people]
-  
-  doc = { name: 'Steve', hobbies: [ 'hiking', 'tennis', 'fly fishing' ] }
-  
+
+  doc = {
+    name: 'Steve',
+    hobbies: [ 'hiking', 'tennis', 'fly fishing' ]
```
Comment (p-mongo) [inline docs/quick-start.txt]: This line needs a comma at the end.

File: docs/quick-start.txt [hunk 2]
```diff
@@ -78,23 +78,34 @@ To insert a single document into a collection, use the
...
+	  siblings: {
```
Comment (p-mongo) [inline docs/quick-start.txt]: This hash is underindented by 1 space it looks like.

**Comment by p-mongo** [comment]
FYI, https://github.com/mongodb/mongoid/pull/4655 did this for Mongoid nested associations.

**Comment by terakilobyte** [comment]
@p-mongo Sounds good about mongoid. The ticket called out these in particular. I've addressed the missing comma and spacing issue.

**Comment by p-mongo** [comment]
Thanks!

**Pull request merged**